.pre-commit-config.yaml

```diff
@@ -1,41 +1,50 @@
+ci:
+  autofix_prs: false
+
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: check-merge-conflict
+      - id: check-case-conflict
+      - id: check-ast
+      - id: debug-statements
+      - id: check-toml
+      - id: check-json
+
+  - repo: https://github.com/pre-commit/pygrep-hooks
+    rev: v1.9.0
+    hooks:
+      # - id: python-check-blanket-noqa # XXX: disallow blanket # noqa
```

Comment:
I've added commented out hooks that I think could benefit the project, but that currently would cause a lot of churn out-of-scope of this initial PR.